http_responses_app/api/fundamentals/client_error.py

```diff
@@ -1,3 +1,17 @@
 from fastapi import APIRouter
-
+from http_responses_app.core.responses import get_response
 router = APIRouter()
+
+data = {}
+
+
+@router.on_event("startup")
+def startup_event():
+    data["quota"] = 3
+    data["limit"] = 0
+
+
+@router.get("/limited")
+async def get_response_403():
+    data["limit"] += 1
```

Comment:
here we have to return just 403, as per documentation `HTTP 403 is returned when the client is not permitted access to the resource despite providing authentication such as insufficient permissions of the authenticated account`
So we have to provide headers, checking for authentication - if no headers then return 401, when header is provided - return 403 with proper message like "For priviledged users only"